ibm/resource_ibm_is_lb_listener.go

```diff
@@ -260,11 +267,17 @@ func lbListenerCreate(d *schema.ResourceData, meta interface{}, lbID, protocol,
 	if err != nil || lb == nil {
 		return fmt.Errorf("Error getting Load Balancer : %s\n%s", err, response)
 	}
-
 	if lb != nil && *lb.RouteMode && lb.Profile != nil && *lb.Profile.Name == "network-fixed" {
 		portMin := int64(1)
 		portMax := int64(65535)
 
+		options.PortMin = &portMin
+		options.PortMax = &portMax
+	} else if lb != nil && lb.Profile != nil && strings.EqualFold(*lb.Profile.Family, "network") {
```

Comment:
API accepts only port as well for network family. Should we have this check to keep this mandatory for network LB ?
if we don't provide port/portmin/portmax the api throws "Listener port or port_min is missing.", 
and it accepts if we provide just port as eg. 443. it sets back 443 for portmin/max.
can we check for "port" as well in the condition ?

Reply:
in this case looks like only one of port, port_min is required by the api

Reply:
ideally user should either use port or range, not both